std/process.d

```diff
@@ -908,6 +908,19 @@ unittest // Reopening the standard streams (issue 13258)
     assert(lines == ["foo", "bar"]);
 }
 
+version (Windows)
```

Comment:
Well, we're testing a 32/64 inconsistency in how C FILE flags carry over to Windows file HANDLEs. FIxing the unit test to work on Posix isn't a problem but I'd be surprised if it failed, because as I understood the append flag is actually getting sent to the OS.